lib/driver.js

```diff
@@ -41,13 +42,35 @@ class WindowsDriver extends BaseDriver {
     this._screenRecorder = null;
   }
 
+  async executePrerun (spec) {
+    if (!_.isPlainObject(spec)) {
+      throw new Error(`'prerun' capability value must be an object`);
+    }
+    const { script, command } = spec;
+    if (_.isString(command)) {
+      this.ensureFeatureEnabled(POWER_SHELL_FEATURE);
```

Comment:
👍

Reply:
If we create a WinAppDriver session directly, without Appium installed on Node.js can we still use this feature?

Reply:
No, this feature runs via appium, before/after create/delete a session.
You should do the same steps in your implementation

Reply:
Thank you very much, understood.